app/components/provider_interface/user_permissions_review_component.rb

```diff
@@ -0,0 +1,32 @@
+module ProviderInterface
+  class UserPermissionsReviewComponent < SummaryListComponent
+    def initialize(permissions, change_path)
+      @permissions = permissions
+      @change_path = change_path
```

Comment:
when we select change from the review page, should we navigate to the highlighted checkbox on the edit page?

Reply:
Just confirmed in a design demo that in this case, for now, we should just link to the edit page with no anchor